src/control.c

```diff
@@ -0,0 +1,41 @@
+#include "control.h"
+
+static volatile float target_linear_speed;
```

Comment:
I would add the units on the variables

Reply:
I'd say could omit the units as long as they follow the International System of Units.